contrib/inventory/ec2.ini

```diff
@@ -112,6 +112,7 @@ group_by_key_pair = True
 group_by_vpc_id = True
 group_by_security_group = True
 group_by_tag_keys = True
+group_by_tag_values = True
```

Comment:
i would default to false as most current users won't expect a bunch of new groups